mhs-reference-implementation/selenium_tests/page_objects/methods.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+from selenium_tests.page_objects import interactions, xml_parser
+
+HUMAN_READABLE = 'Payload stuff'
+
+
+def __init__(self, driver):
+    self.driver = driver
+
+
+def get_asid():
+    ###The asid should be stored in data\local_asid file (which is excluded from GIT
+    with open(os.path.join(os.getcwd(), "selenium_tests\\data\\local_asid")) as asid_file:
```

Comment:
I think this line could be better as:
```suggestion
    with (Path(ROOT_DIR) / "selenium_tests" / "data" / "local_asid").open() as asid_file:
```

or instead of the whole with statement, you could do [`asid = (Path('blah' / ...).read_text()`](https://github.com/python/cpython/blob/d5c120f7eb6f2a9cdab282a5d588afed307a23df/Lib/pathlib.py#L1204) though that might include a newline on the end.